bundle/src/main/java/com/adobe/acs/commons/data/Spreadsheet.java

```diff
@@ -229,7 +230,7 @@ public String convertHeaderName(String str) {
             if (str.contains("@")) {
                 str = StringUtils.substringBefore(str, "@");
             }
-            return String.valueOf(str).toLowerCase().replaceAll("[^0-9a-zA-Z:\\-]+", "_");
+            return String.valueOf(str).replaceAll("[^0-9a-zA-Z:\\-]+", "_");
```

Comment:
There are a number of methods on JcrUtil for creating valid JCR names.. Could/should one of these be used instead of this more bespoke (regex) solution?

https://helpx.adobe.com/experience-manager/6-2/sites/developing/using/reference-materials/javadoc/com/day/cq/commons/jcr/JcrUtil.html

Reply:
Also, perhaps a quick unit test or two (esp if we have known failing cases which i assume drove this update?)